tests/tools/include/detray/options/parse_options.hpp

```diff
@@ -8,7 +8,10 @@
 #pragma once
 
 // Boost
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wnull-dereference"
 #include <boost/program_options.hpp>
+#pragma GCC diagnostic pop
```

Comment:
You could consider wrapping this header in a custom one which includes the `pragm`.

Reply:
Done